modules/react-native-status/android/src/main/java/im/status/ethereum/pushnotifications/NewMessageSignalHandler.java

```diff
@@ -122,9 +125,15 @@ public Intent getOpenAppIntent() {
     //NOTE: this method takes a chatId and returns an intent that will open the app in that chat
     //Once we support other kind of notifications we will need to adapt it. The simplest method
     //is probably to pass the universal link as param instead of the chatId.
-    public Intent getOpenAppIntent(String chatId) {
+    public Intent getOpenAppIntent(String chatId, int chatType) {
         Intent intent = getOpenAppIntent();
-        intent.setData(Uri.parse("status-im://p/" + chatId));
+        String path = "";
+        if (chatType == 1) {
```

Comment:
Probably best to put this in some constant

Reply:
fixed